docs/upgrade/upgrade-security.asciidoc

```diff
@@ -1,44 +1,77 @@
 
+NOTE: For detailed information about the 8.0.0 release, check out the <<release-notes-8.0.0, Release notes>>.
 
 [discrete]
-[[pre-upgrade-security]]
-== Pre-upgrade requirements for 8.0
+[[upgrade-reqs]]
+== Upgrade requirements
+
+*Track rules that are automatically disabled when upgrading*
+
+Upon upgrading to 8.0.0, rules are automatically disabled. Once the upgrade completes, you should re-enable them again to avoid gaps in rule coverage. _We highly recommend that you track your active rules before upgrading so you can easily find and re-enable them after upgrading to 8.0.0_.
+
+Before upgrading, use the <<rules-api-find, Find rules API>> to capture a list of enabled rules.
+
+IMPORTANT: Use curl or another HTTP tool to run {elastic-sec} API commands.
+
+Here is a curl command that you can run to find enabled rules:
+
+[source,console]
+--------------------------------------------------
+GET /api/detection_engine/rules/_find?per_page=10000&filter=alert.attributes.enabled:true
+--------------------------------------------------
+
+To re-enable your rules from the Rules page:
+
+. Go to the All rules table (*Detect -> Rules*).
+. Select the rules that you want to enable.
+. Click *Bulk actions -> Activate Selected* to re-enable the rules.
+
+Alternatively, you can use the <<bulk-actions-rules-api, Bulk rule actions>> API to re-enable rules.
+
+
+*New Full Disk Access approval needed for {elastic-endpoint}*
+
+When you manually install {elastic-endpoint}, you must approve a system extension, kernel extension, and enable Full Disk Access (FDA). There is a new FDA requirement in 8.0.0. Refer to <<deploy-elastic-endpoint>> to review the required permissions.
 
-Before you upgrade, review the following requirements specific to {elastic-sec}:
 
 *Upgrade agents*
 
-Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). After this is complete, you can then {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0. Initially, {agent}s will be version 7.17; this is fine because {elastic-sec} supports the last minor in 7.x (7.17) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0.0, which is recommended to ensure you get the latest features and improvements.
+Upgrade your {stack} and {agent}s to 7.17.0 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). After this is complete, you can then {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0. Initially, {agent}s will be version 7.17.0; this is fine because {elastic-sec} supports the last minor in 7.x (7.17.0) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0.0, which is recommended to ensure you get the latest features and improvements.
```

Comment:
```suggestion
Upgrade your {stack} and {agent}s to 7.17.0 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). After this is complete, you can then {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0.0. Initially, {agent}s will be version 7.17.0; this is fine because {elastic-sec} supports the last minor release in 7.x (7.17.0) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0.0, which is recommended to ensure you get the latest features and improvements.
```